src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+  isLoading: true,
+};
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState,
+});
+
+export default booksSlice.reducer;
```

Comment:
- [ ] Based on project requirements, you're meant to creates 2 reducers, `addBook` and `removeBook`
- [ ] You're also meant to exports the reducers:  `addBook` and `removeBook` as actions. Kindly implement both 👍 

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/230340006-ed31e49e-e77a-427c-991f-7e7cf2fc7c4c.png)